src/components/Contacts/ContactDetails/ContactDetailsTab/Other/ContactDetailsOther.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, CssBaseline, MuiThemeProvider } from '@material-ui/core';
+import React, { ReactElement } from 'react';
+import { gqlMock } from '../../../../../../__tests__/util/graphqlMocking';
+import theme from '../../../../../theme';
+import {
+  ContactDetailsTabDocument,
+  ContactDetailsTabQuery,
+} from '../ContactDetailsTab.generated';
+import { ContactDetailsOther } from './ContactDetailsOther';
+
+export default {
+  title: 'Contacts/Tab/ContactDetailsTab/Other',
+  component: ContactDetailsOther,
+};
+
+const accountListId = '111';
+const contactId = '222';
+
+export const Default = (): ReactElement => {
+  const mock = gqlMock<ContactDetailsTabQuery>(ContactDetailsTabDocument, {
+    variables: {
+      accountListId: accountListId,
```

Comment:
Not sure why this is failing. the check.

Reply:
Looked into this and turns out the default type for the variables is ```never```. See https://github.com/CruGlobal/mpdx-react/blob/9c2102b7d4a45216fd074408b09aaf31815254ae/__tests__/util/graphqlMocking.tsx#L133
So to fix this, I just included the generated types for the query variables.